alerts/storage_alerts.libsonnet

```diff
@@ -51,6 +51,41 @@
               message: 'Based on recent sampling, the PersistentVolume claimed by {{ $labels.persistentvolumeclaim }} in Namespace {{ $labels.namespace }} is expected to fill up within four days. Currently {{ $value | humanizePercentage }} is available.',
             },
           },
+          {
+            alert: 'KubePersistentVolumeRunningOutOfInodes',
+            expr: |||
+              kubelet_volume_stats_inodes_free{%(prefixedNamespaceSelector)s%(kubeletSelector)s}
+                /
+              kubelet_volume_stats_inodes{%(prefixedNamespaceSelector)s%(kubeletSelector)s}
+                < 0.03
+            ||| % $._config,
+            'for': '1m',
+            labels: {
+              severity: 'critical',
```

Comment:
I'm not sure we should have the critical one at all.
`severity: warning` is totally fine :+1: I'm a bit concerned, cause there might be some application that's running totally fine with less than `0.03` inodes left.
This being a cause and not even a symptom and then paging to wake up people is probably too much. If it really becomes a problem, the alerting on your actual application should page and wake one up.

Reply:
We do have the same alerting structure for [running out of space](https://github.com/kubernetes-monitoring/kubernetes-mixin/blob/master/alerts/storage_alerts.libsonnet#L19-L34) and this is also [accepted pattern in node_exporter mixin](https://github.com/kubernetes-monitoring/kubernetes-mixin/blob/master/alerts/storage_alerts.libsonnet#L19-L34). I think we should be consistent and either reduce severity here and in `KubePersistentVolumeFillingUp` or accept that running out of space/inodes is one of the exceptions where alerting on cause is something that is excused due to lack of good symptom.

Reply:
Ok, I guess I can live with the fact to have this for the sake of consistency. It doesn't mean I'm entirely keen on the idea. :upside_down_face: In this case, feel free to go ahead and ignore my comment. :grin: